docs/specifications/tests/Zone-TP/zone11.md

```diff
@@ -0,0 +1,201 @@
+# ZONE11: SPF policy validation
+
+## Test case identifier
+**Zone11**
+
+## Table of contents
+
+* [Objective](#objective)
+* [Caveat](#caveat)
+* [Scope](#scope)
+* [Inputs](#inputs)
+* [Summary](#summary)
+* [Test procedure](#test-procedure)
+* [Outcome(s)](#outcomes)
+* [Special procedural requirements](#special-procedural-requirements)
+* [Intercase dependencies](#intercase-dependencies)
+* [Terminology](#terminology)
+
+## Objective
+
+Sender Policy Framework (SPF), described in [RFC 7208][RFC7208], is a mechanism
+allowing domain name owners to specify which hosts are allowed to send mail
+claiming to be from that domain. It is implemented by means of TXT records in a
+structured format.
+
+This test case looks up SPF records in the domain to be tested. It checks that
```

Comment:
In the apex of the domain to be tested?

The the zone is name.xa do you put the SPF records in "name.xa. TXT" to apply for "xxx@name.xa"? Do they also apply for "xxx@subname.name.xa"?